sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/EventProcessorOptions.cs

```diff
@@ -87,7 +87,7 @@ public bool EnableReceiverRuntimeMetric
         public Func<string, EventPosition> InitialOffsetProvider { get; set; }
 
         /// <summary>
-        /// Returns whether the EventProcessorHost will call IEventProcessor.OnEvents(null) when a receive
+        /// Returns whether the EventProcessorHost will call <c>IEventProcessor.ProcessEventsAsync(PartitionContext context, Enumerable.Empty<EventData>())</c>when a receive
```

Comment:
@jsquire How about this?

```suggestion
        /// Returns whether the EventProcessorHost will call IEventProcessor.ProcessEventsAsync(context, Enumerable.Empty<EventData>()) when a receive
```